packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -213,7 +196,7 @@ async function createWebLoader(
 
     await codeLoader.seedModule(
         codeDetails,
-        wrapWithRuntimeFactoryIfNeeded(codeDetails.package as IFluidPackage, fluidModule),
+        addFakeDetailsIfNeeded(codeDetails.package as IFluidPackage, fluidModule),
```

Comment:
Probably would be better for WebCodeLoader to apply its own details to the module (as if it had done the load) as part of the seedModule() call rather than us applying the details here, but I think that would be technically a breaking change.